test/AutoDiff/leakchecking.swift

```diff
@@ -72,7 +72,7 @@ LeakCheckingTests.test("BasicVarLeakChecking") {
   }
 
   // TODO: Fix memory leak.
-  testWithLeakChecking(expectedLeakCount: 1) {
+  testWithLeakChecking(expectedLeakCount: 0) {
```

Comment:
```suggestion
  testWithLeakChecking {
```
`expectedLeakCount` argument has default value 0.